packages/plugin-coinbase/src/utils.ts

```diff
@@ -380,22 +380,24 @@ export async function executeTransfer(wallet: Wallet, amount: number, sourceAsse
 
 /**
  * Gets the charity address based on the network.
- * For now we are giving to the following charity, but will make this configurable in the future
- * https://www.givedirectly.org/crypto/?_gl=1*va5e6k*_gcl_au*MTM1NDUzNTk5Mi4xNzMzMDczNjA3*_ga*OTIwMDMwNTMwLjE3MzMwNzM2MDg.*_ga_GV8XF9FJ16*MTczMzA3MzYwNy4xLjEuMTczMzA3MzYyMi40NS4wLjA.
  * @param {string} network - The network to use.
+ * @param {boolean} isCharitable - Whether charity donations are enabled
+ * @throws {Error} If charity address for the network is not configured when charity is enabled
  */
-export function getCharityAddress(network: string): string {
- let charityAddress;
-    if (network === "base") {
-        charityAddress = "0x1234567890123456789012345678901234567890";
-    } else if (network === "sol") {
-        charityAddress = "pWvDXKu6CpbKKvKQkZvDA66hgsTB6X2AgFxksYogHLV";
-    } else if (network === "eth") {
-        charityAddress = "0x750EF1D7a0b4Ab1c97B7A623D7917CcEb5ea779C";
-    } else if (network === "arb") {
-        charityAddress = "0x1234567890123456789012345678901234567890";
-    } else if (network === "pol") {
-        charityAddress = "0x1234567890123456789012345678901234567890";
+export function getCharityAddress(network: string, isCharitable: boolean = false): string | null {
```

Comment:
We will also want to handle what happens when getCharityAddress returns null. I had some changes here for that feel free to take a look and integrate: https://github.com/ai16z/eliza/pull/801/commits/f7ace735eae682b4a2476a44174c73c080bf5077

Reply:
its a consensus of the group to have charitble as a boolean set to false